docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +160,11 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-automatically]]
+=== Building the OpenShift Documentation Automatically
```

Comment:
"Building the OpenShift Documentation using CentOS CI"

'Automatically' is really ambiguous and not really what this is about